style.css

```diff
@@ -52,6 +52,22 @@ h1, h2 {
 			background: #f06;
 		}
 
+		.w3c-link::before,
+		.mdn-link::before {
+			content:'';
+			display: inline-block;
+			height: 16px;
+			width: 16px;
+			background: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAMAAABEpIrGAAAAqFBMVEUAAAD///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////8j1z1tAAAAOHRSTlMA9eXYekrw6rZkJOHewKuhnXQbEgbu0cu6qIR/W1U7NDEfCwLtyJWPb05BGA0Iw7uvpIwrBNVsKu/y7/oAAAFHSURBVDjL3ZFXcsMwDAWxVLWam9ziuJfELb3c/2aByFjyGbw/bzCzQ4CA3AtjU1TRMlli65Nn/omd8MG2p/EJLVvPuLJ3whFmGpsdb7buYPxut2swPSesMzpVTjF9jYtnyyDkWf5p4601HiDQmNsHzx6jy1V4hLnI1wietPKJTvqcsx29iKkOGcKwFMnR4ZOI9u1HeRF5ZhuykNVAe28m4JeN0GWwkph9jl/1W8oCJV/VwhIe1xHzKa868fBLks4kg936KpRD9gFpOSf6/dZJK/qFW5zjieEbEzmntEOCevS4FgKw/rtG2hdHjFcL/RQGid0IRbM/IzUF5Bqr7U3jmKwRWuBX+UqYbA7FsnrVkDdCojtyG3mRzYjx2Z7Gl4ZddnEbeRA5wMjvYM9d8/Fuo/SO7q7KIJAbjgeXPydRZuPUmyzkfvgDdPETe8fVD1QAAAAASUVORK5CYII=') 0 0 / 16px 16px;
```

Comment:
Why a PNG? An SVG would be much smaller, and wouldn't need to be base64 encoded.

Reply:
I have no SVG. I will try to food it.